Core/Security/ProxyInfo.py

```diff
@@ -17,24 +17,26 @@
 
 def getProxyInfo( proxy = False, disableVOMS = False ):
   """
-  Returns a dict with all the proxy info
-  * values that will be there always
-   'chain' : chain object containing the proxy
-   'subject' : subject of the proxy
-   'issuer' : issuer of the proxy
-   'isProxy' : bool
-   'isLimitedProxy' : bool
-   'validDN' : Valid DN in DIRAC
-   'validGroup' : Valid Group in DIRAC
-   'secondsLeft' : Seconds left
-  * values that can be there
-   'path' : path to the file,
-   'group' : DIRAC group
-   'groupProperties' : Properties that apply to the DIRAC Group
-   'username' : DIRAC username
-   'identity' : DN that generated the proxy
-   'hostname' : DIRAC host nickname
-   'VOMS'
+  :Returns: a dict with all the proxy info:
```

Comment:
No this works: http://diracsailer.readthedocs.io/en/fixdoc/CodeDocumentation/Core/Security/ProxyInfo.html#DIRAC.Core.Security.ProxyInfo.getProxyInfo
`:Returns:`, `:returns:`, `:return:` all the same

http://www.sphinx-doc.org/en/1.4.9/domains.html#info-field-lists